### PR TITLE
Fix tootip overlay closing.

### PR DESCRIPTION
**Pull request by natacha-beck**
Multiple little feature: 

- Unable the closing functionality when leaving `html_tool_tip`.
- Add a `close` at the bottom of the `html_tool_tip`
- Unable the opening of a `html_tool_tip` when mouseenter only if no other are open. 
- It is now possible to open multiple `html_tool_tip` only if click on the link instead `mouseenter`

### CODE REVIEW

**Comment by MontrealSergiy** [comment]
Can you refer the issue(s) it is using by id or in Development section?

**Comment by natacha-beck** [comment]
Fix the https://github.com/aces/cbrain/issues/1433

**Review by MontrealSergiy — APPROVED** [review]
I do not like that same control ( underlined "more") behaves differently on different pages.
But the PR address the issue as requested.

Maybe we can rename it into 'info', 'card', 'infocard' or '+' ?

**Comment by MontrealSergiy** [comment]
The request recommends to use logic more similar to task help, but the PR still resolves the issue, even though behaviour of 'more' control diverges from the rest of cbrain.

Perhaps, for the case of multiple windows I would add somewhere a button "close all info-cards"

Screenshot from 2024-10-28 14-41-16

**Comment by natacha-beck** [comment]
> I do not like that same control ( underlined "more") behaves differently on different pages. But the PR address the issue as requested.
> 
> Maybe we can rename it into 'info', 'card', 'infocard' or '+' ?

Can you let me know which pageS you are referring to ?

**Comment by MontrealSergiy** [comment]
for example, https://portal.cbrain.mcgill.ca/tools

**Comment by natacha-beck** [comment]
It have the same behaviour on both page ( i just tested, let me know if it is not the case). 

Also I look in the CBRAIN portal and notice some strange behaviour on the tool page:

image

Will look at it,

**Comment by MontrealSergiy** [comment]
Ok, you right. I guess I just do not like that more element.

File: BrainPortal/app/controllers/nh_users_controller.rb [hunk 1]
```diff
@@ -87,9 +87,9 @@ def update
     attr_to_update.delete(:zenodo_sandbox_token) if attr_to_update[:zenodo_sandbox_token].blank?
     attr_to_update.delete(:zenodo_main_token)    if attr_to_update[:zenodo_main_token].blank?
 
-    # Do not update password if user must use globus
-    if user_must_link_to_oidc?(@user)
-      flash[:error] = "You cannot change the password for your account." if attr_to_update[:password].present?
+    # Do not update password if user must use globus (or other oidc)
+    if user_must_link_to_oidc?(@user) && attr_to_update[:password].present?
+      flash[:error] = "You cannot change the password for your account because you should use OpenID." if attr_to_update[:password].present?
```
Comment (MontrealSergiy) [inline BrainPortal/app/controllers/nh_users_controller.rb:92]: not sure how OpenID relates to tootips

**Comment by natacha-beck** [comment]
Replace by #1449

**Comment by natacha-beck** [comment]
Replace by #1449

**Pull request closed without merging**